package.json

```diff
@@ -3,5 +3,9 @@
   "private": true,
   "devDependencies": {
     "lerna": "^3.20.2"
+  },
+  "scripts": {
+    "init": "yarn && lerna bootstrap --force-local && lerna link",
```

Comment:
I believe that `lerna bootstrap` performs a dependency symlinking. Do we need an additional`lerna link` here?

Reply:
@nathanstilwell the first two commits are actually already in master. I asked the same question https://github.com/cockroachdb/ui/pull/44#discussion_r420134062

Also added a note as part of the README ticket in case we want to adjust anything https://github.com/cockroachdb/ui/issues/36#issuecomment-624882442

Reply:
Thanks for the head's up @laurenbarker.